File: src/gestionMain/gestionCliente/RegistroCliente.java
package gestionMain.gestionCliente;

import java.util.Scanner;

import gestionApp.Pedido.Cliente;



public class RegistroCliente {
    public static void registrarCliente() {
        int opcion;
        Scanner scan = new Scanner(System.in);

        do {
            System.out.print("\tIngrese los siguiente campos ");
            System.out.print("\nCedula: ");
            String cedula = scan.nextLine();
            System.out.print("Nombre Completo: ");
            String nombre = scan.nextLine();
            System.out.print("Celular: ");
            int cel = scan.nextInt();
            Cliente.agregarCliente(new Cliente(cedula, nombre, cel));
            System.out.println("Sus datos de han guardado satisfactoriamente");
            System.out.println("Eres el numero de cliente: "+Cliente.getNumeroCliente());
            System.out.print("1. Regresar : ");
            opcion = scan.nextInt();
        } while (opcion != 1);

    }

}

File: practica-2-grupo-2-equipo-4/GestionMain/gestionCliente/RegistroCliente.py
class RegistroCliente:
    
    def registrarCliente(opcion):
         while opcion != 2:
            try:
                print("\tIngrese los siguiente campos")
                cedula = int(input("\nCedula: "))
                nombre = str(input("\nNombre Completo: "))
                celular = int(input("\nCelular: "))
                # Agregar Cliente
                
                opcion = int(input("\n1. Para modificar algun dato \n2. Para Continuar con la Orden \n"))
                print("Sus datos de han guardado satisfactoriamente")
            except :
                print("Ingrese una opcion nuevamente: ")
                break
